docs/source-2.0/tutorials/full-stack-tutorial.rst

```diff
@@ -0,0 +1,793 @@
+======================
+Full Stack Application
+======================
+
+Overview
+========
+In this tutorial, we’ll walk you through using Smithy in a full-stack application. This includes defining a model for
+a simple coffee-shop service, generating TypeScript code for both the client and server, and implementing both a
```

Comment:
```suggestion
a simple coffee shop service, generating TypeScript code for both the client and server, and implementing both a
```

Reply:
We dont really say what a "coffee shop" service does. Would be good to have a bit more background. Why are we building a coffee shop API? What do we expect to be able to do?

Reply:
Added as the first paragraph. Wondering if we should move it somewhere else though - maybe under `Modelling Our Service`. Thoughts?